application/controllers/resetpassword.php

```diff
@@ -0,0 +1,30 @@
+<?php 
+
+session_start();
+require_once './vendor/autoload.php';
+
+use Database\Database;
+use User\User;
+
+$Object_database = new Database();
+$Object_user = new User;
+if(isset($_POST['submit'])){
+  if($Object_user->Validate_Password($_POST['password'])){
+    $Object_user->update_password($_SESSION['mail'],$_POST['password']);
+    session_destroy();
+    $GLOBALS['updated'] = true;
+    $GLOBALS['success'] = "Password Updated";
+    require_once './application/views/login.php';
+  }
+  else {
+    $GLOBALS['updated'] = false;
```

Comment:
If user once try to reset the password and user entered invalid password then in that case `session` will be not destroy! Please check it.

Reply:
Yes correct. I will add destroy the session inside the else condition also